### ISSUE TITLE
Cannot generate ICS when Location Field is defined

### ISSUE THREAD

**Issue by MK-PL** [verbb/Events#80]
**Description**

I report a bug in the plugin - the link to ICS will not be generated (it throw error as in the screenshot) when the Location field is defined. When this field is left blank, it works fine.

**Steps to reproduce**
1. Set up Location Field in Event Type settings.
2. Click on generated ICS link.

**Additional info**
- Plugin version: 1.4.15
- Craft version: Craft Pro 3.5.9
- Multi-site: Yes

**Screenshots**
![image](https://user-images.githubusercontent.com/8502239/115865269-6421e980-a438-11eb-9c09-b3526875ac32.png)

![image](https://user-images.githubusercontent.com/8502239/115865095-22913e80-a438-11eb-8cab-bf0ba9ed2f71.png)

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/events` requirement in `composer.json` to:

```
"require": {
  "verbb/events": "dev-craft-3 as 1.4.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.16](https://github.com/verbb/events/releases/tag/1.4.16)
